var/spack/repos/builtin/packages/seissol/package.py

```diff
@@ -206,8 +210,11 @@ class Seissol(CMakePackage, CudaPackage, ROCmPackage):
 
     depends_on("asagi +mpi +mpi3", when="+asagi")
 
-    depends_on("easi ~asagi jit=impalajit,lua", when="~asagi")
-    depends_on("easi +asagi jit=impalajit,lua", when="+asagi")
+    depends_on("easi ~asagi jit=lua", when="~asagi")
+    depends_on("easi +asagi jit=lua", when="+asagi")
+
+    depends_on("easi@:1.4 ~asagi jit=impalajit,lua", when="~asagi ^easi@:1.4")
```

Comment:
Maybe just `easi` will work here for the tests

Reply:
I don't understand?
(maybe you mean, I could, just remove these 2 lines? if people really want to use older version they could put it manually, e.g. with 
`spack install seissol ^easi@1.2.0 jit=impalajit,lua + asagi`?)

Reply:
My spack knowledge isn't that good; but I was wondering if you could fix the tests by setting `depends_on("easi ~asagi jit=impalajit,lua", when="~asagi ^easi@:1.4")` and similar.

Reply:
ha ok I did not see the unittest failing. thanks